metadata-suggestions.md

```diff
@@ -0,0 +1,528 @@
+# Metadata suggestions from live import
+[Back to the index](README.md)
+
+**Contents:**
+
+- [Introduction](#introduction)
+- [Main Endpoint](#main-endpoint)
+- [Single suggestion endpoint](#single-suggestion-endpoint)
+- [Linked entities](#linked-entities)
+	- [External source entries](#external-source-entries)
+	- [Single entry](#single-entry)
+	- [Changes for a single entry](#changes-for-a-single-entry)
+- [Changes suggested from the external source](#changes-suggested-from-the-external-source)
+	- [Introduction](#introduction)
+	- [Adding metadata](#adding-metadata)
+	- [Replacing metadata](#replacing-metadata)
+
+## Introduction
+
+This contract allows for suggesting metadata changes to an in-submission or in-workflow item.
+The user can choose an external source to use for suggesting the metadata (e.g. PubMed for a medical publication, ORCID for an author, a RIS file for uploaded metadata).
+The external source will suggest matches, and will suggest metadata changes if the match is accepted by the user.
+A full metadata description of the data from the external source is displayed in combination to the suggested changes to the current item.
+
+TODO:
+* start a new submission based on a external source record
+
+## Main Endpoint
+**/api/integration/metadata-suggestions**
```

Comment:
I did write the introduction at the top of the page to explain this in detail.
Maybe it's more clear if you first review the actual features from https://github.com/atmire/Rest7Contract/blob/metadata-suggestions/metadata-suggestions.md#changes-suggested-from-the-external-source

Suggestions always start from an existing item which can have metadata and bitstreams attached to it. It needs the item in order to offer suggestions. It's tailored to creating various types of suggestions explained in https://github.com/atmire/Rest7Contract/blob/metadata-suggestions/metadata-suggestions.md#single-suggestion-endpoint

The overlap with external sources is that both solutions can search in various sources and the sources can overlap. This will also be reflected in the implementation where the overlapping parts would of course only be implemented once. There's no need to have two implementations for e.g. ORCID in the backend, just a configuration whether ORCID will be used as an external source and/or as a live import.

Reply:
@benbosman : I have to admit, I'm still a bit confused by the mixed terminology here. This header says "Changes Suggested from the External Source": https://github.com/atmire/Rest7Contract/blob/metadata-suggestions/metadata-suggestions.md#changes-suggested-from-the-external-source   Does "external source" in that header refer to the same external sources returned from `/api/integration/externalsources`? https://github.com/DSpace/Rest7Contract/blob/master/external-authority-sources.md

So, are these endpoints directly related, or are they entirely separate?  They sound separate to me, but then it's confusing to use the phrase "external source" in two different ways.

Reply:
@tdonohue I've tried to adjust the wording to avoid any confusion.
The endpoints are indeed separate. They can have e.g. both offer ORCID, but from a different perspective (for importing an ORCID record in authority control or related item vs for importing an ORCID record in the current item)

Let me know if you see any other sections where this difference is not clear yet

Reply:
Resolving this conversation. Differences between this and External Sources were discussed in greater detail in this meeting: https://wiki.duraspace.org/display/DSPACE/2019-11-05+DSpace+7+Entities+WG+Meeting#id-2019-11-05DSpace7EntitiesWGMeeting-Notes